src/utils/io/mmap.rs

```diff
@@ -113,7 +113,10 @@ mod tests {
 
     #[test]
     fn test_out_of_band_mmap_read() {
-        let temp_file = tempfile::NamedTempFile::new_in(".").unwrap();
+        let temp_file = tempfile::Builder::new()
+            .tempfile()
+            .unwrap()
+            .into_temp_path();
```

Comment:
nit:
```suggestion
        let temp_file = tempfile::NamedTempFile::new()
            .unwrap()
            .into_temp_path();
```

same for all the others

Reply:
https://github.com/ChainSafe/forest/blob/no-test-artifacts-in-root/.clippy.toml#L23-L25